### DISCUSSION TITLE
anti-tab complete with bypass permission and command suggest whitelist

### DISCUSSION THREAD

**Discussion by ArtifactyNight** [Zrips/CMI#2918] [Feature Requests]
**Description of feature/improvement request:**
+ add "anti-tab complete" with bypass permission `cmi.bypass.tabcomplete`

+ add command suggestion whitelist, example:
   if enable blocked tab-complete features, it's will can using this feature
   .. to whitelist suggest command, it's will add this in config.yml
   
```
  filter-commands:
    - spawn
    - tpa
    - etc.
```

so.. player can only see filter command if `/[TAB]`

sorry for bad grammar

---
### Detailed explanation of what it should do, when and in what situation: ###
in details
### Basic explanation as to why this feature is needed and/or useful to have: ###
can limit command suggest when /[TAB], so i dont want player to see other commands x)

**Comment by glorying16** [comment]
There was a plugin like this for a bit, but the dev (Gasha) was removed from spigot. It was called Anti-Command Suggestions. It is open source but I unfortunately dont have a clue what to do with it. But I would LOVE for CMI to take it on. 

Ill add to what NightPrimez said by asking that a potential feature like this be in permission groupings. Like whitelistA would correspond to permissionA, and whitelistB to permissionB, etc. This would make it easier for lower level players to understand their usable permissions.

**Comment by Insprill** [comment]
PL-Hide should do the trick for ya :P
